diff --git a/src/headers.rs b/src/headers.rs
index 8407be18..ad29ad28 100644
--- a/src/headers.rs
+++ b/src/headers.rs
@@ -53,7 +53,7 @@ pub(super) fn content_length_parse_all_values(values: ValueIter<'_, HeaderValue>
                         return None;
                     }
                 } else {
-                    return None
+                    return Some(0_0);
                 }
             }
         } else {
            